src/test/Drips.t.sol

```diff
@@ -703,6 +703,25 @@ contract DripsTest is DSTest {
         receiveDrips(receiver, cycleSecs * 2);
     }
 
+    function testUpdateDefaultStartDrip() public {
```

Comment:
@CodeSandwich 
This testcase fails we init `start = startCap;` instead of `start = updateTime;`